idl/cherami_server/shared.thrift

```diff
@@ -379,3 +392,8 @@ struct ListExtentsStatsResult {
   1: optional list<ExtentStats> extentStatsList
  10: optional binary nextPageToken
 }
+
+struct SealExtentRequest {
+  1: optional string extentUUID
+  2: optional i64 (js.type = "Long") sequenceNumber
```

Comment:
Hmm.. The sequence number really doesn't make sense for this call from controller to input-host. I know I voted for the name "SealExtent" on the input host (for consistency), but reusing the same request/resp structs makes this API even more confusing. 

I kinda like your idea now, of just naming this API as admin,drainExtent(extentUUID). This is all assuming the controller will do the sealing and input will just drain the extent. What do you think ?